sniffles/feature.py

```diff
@@ -39,15 +39,7 @@ def __str__(self):
 
     def toString(self):
         num_elements = random.randint(1, self.max_list_size)
-        myelements = self.values
-        random.shuffle(myelements)
-        myString = "["
-        for index, elem in enumerate(myelements[0:num_elements]):
-            myString += elem
-            if index != num_elements - 1:
-                myString += ","
-        myString += "]"
-        return myString
+        return '[' + ','.join(random.sample(self.values, num_elements)) + ']'
```

Comment:
Will this cause the last element in the list to have a comma?  If so, is that problematic?  I purposely avoided that in the original code, but I am not certain that it really matters.

Reply:
No, it won't.
```
In [3]: ','.join(['a', 'b', 'c'])
Out[3]: 'a,b,c'
```

Reply:
Wow.  Join is smarter than I thought.  Excellent.